src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.ConverterDataToObjImpl;
+import core.basesyntax.service.ReportWriterToFileImpl;
+import core.basesyntax.service.TransactionStrategy;
+
+public class Main {
+
+    public static void main(String[] args) {
+        final String filePath = "src/main/resources/data.CSV";
+        final String reportPath = "src/main/resources/report.CSV";
+        ConverterDataToObjImpl converterDataToObj = new ConverterDataToObjImpl();
+        TransactionStrategy transactionStrategy = new TransactionStrategy();
+        ReportWriterToFileImpl reportWriterToFile = new ReportWriterToFileImpl();
+
+        converterDataToObj.convertData(filePath);
+        transactionStrategy.getShopService();
+        reportWriterToFile.getReport(reportPath);
```

Comment:
Implement main() according to the requirements.
![image](https://github.com/mate-academy/jv-fruit-shop/assets/20326858/abb11c8e-41b5-4081-bf1d-743e620c4b9d)